src/core/Flora/Environment.hs

```diff
@@ -64,13 +68,33 @@ mkPool connectionInfo timeout' connections =
         (realToFrac timeout')
         connections
 
+data BlobStoreImpl = BlobStoreFS FilePath | BlobStorePure
+  deriving stock (Generic, Show)
+
+instance ToJSON BlobStoreImpl
+
+newtype FeatureEnv = FeatureEnv {blobStoreImpl :: Maybe BlobStoreImpl}
```

Comment:
I think it should go to the term level, within a type that declares the operational mode of Flora: meta-index or stand-alone repository. I'm available if you want to chat more about this